### PR TITLE
update event edit form

### PR DESCRIPTION
**Pull request by gayeonsarahyoo**
- updated the edit form to display two separate text fields for bootcamp meeting/resource links.
- two buttons will be displayed on the event card: Meeting and Resources.

### CODE REVIEW

File: frontend/src/components/Home Page/index.js [hunk 1]
```diff
@@ -138,7 +138,7 @@ const Join = () => {return (
             </Col>
 
         </Row> */}
-        {/* <Row> 
+        <Row> 
         <Col sm={6}>
             <h3>Joining with Venmo</h3> 
             <ol>
```
Comment (neerajalavala) [inline frontend/src/components/Home Page/index.js:144]: What did you change here and what is it doing?

File: frontend/src/components/Home Page/index.js [hunk 2]
```diff
@@ -301,7 +301,7 @@ export default class HomePage extends React.Component {
                     <About />
                     <Projects />
                     <Meetings />
-                    {/* <Join/> */}
```
Comment (neerajalavala) [inline frontend/src/components/Home Page/index.js:304]: Same thing

File: frontend/src/components/Signup Page/index.js [hunk 3]
```diff
@@ -146,7 +146,7 @@ class SignUpForm extends React.Component {
                     {/* TODO: REMOVE THIS AND MOVE TO SECONDARY STAGE!!! 
                         This is where the user can select which payment method they prefer. 
                     */}
-                    {/* <Row>
```
Comment (neerajalavala) [inline frontend/src/components/Signup Page/index.js:149]: quick explanation

Comment (gayeonsarahyoo) [reply frontend/src/components/Signup Page/index.js:149]: I think these changes appeared when I tried to merge my stuff to the latest dev branch. They aren't part of the changes I specifically made to update the edit form

**Review by neerajalavala — APPROVED** [review]
looks good to me, lets see how it works

**Pull request merged**